sdk/core/core-http/test/policies/bearerTokenAuthenticationPolicyTests.ts

```diff
@@ -59,18 +59,18 @@ describe("BearerTokenAuthenticationPolicy", function() {
     const refreshCred2 = new MockRefreshAzureCredential(now + TokenRefreshBufferMs);
     const notRefreshCred1 = new MockRefreshAzureCredential(now + TokenRefreshBufferMs + 5000);
 
-    const credentialsToTest: [MockRefreshAzureCredential, number][] = [
-      [refreshCred1, 2],
-      [refreshCred2, 2],
-      [notRefreshCred1, 2]
+    const credentialsToTest: [MockRefreshAzureCredential, number, string][] = [
+      [refreshCred1, 2, "Refresh credential 1"],
+      [refreshCred2, 2, "Refresh credential 2"],
+      [notRefreshCred1, 1, "Not refresh credential 1"]
```

Comment:
In retrospective, the "notRefreshCred1" should haven't refreshed!

Reply:
Original change: https://github.com/Azure/azure-sdk-for-js/pull/10085/files#r472566339

Reply:
I also added these string comments (visible in the diff) to help me debug which case failed.